python/cudf/cudf/tests/test_column.py

```diff
@@ -379,29 +408,22 @@ def test_as_column_buffer(data, expected):
 
 
 @pytest.mark.parametrize(
-    "data,expected",
+    "data,pyarrow_kwargs,cudf_kwargs",
     [
         (
-            pa.array([100, 200, 300], type=pa.decimal128(3)),
-            cudf.core.column.as_column(
-                [100, 200, 300], dtype=cudf.core.dtypes.Decimal128Dtype(3, 0)
-            ),
-        ),
-        (
-            pa.array([{"a": 1, "b": 3}, {"c": 2, "d": 4}]),
-            cudf.core.column.as_column([{"a": 1, "b": 3}, {"c": 2, "d": 4}]),
-        ),
-        (
-            pa.array([[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]]),
-            cudf.core.column.as_column(
-                [[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]]
-            ),
+            [100, 200, 300],
+            dict(type=pa.decimal128(3)),
+            dict(dtype=cudf.core.dtypes.Decimal128Dtype(3, 0)),
         ),
+        ([{"a": 1, "b": 3}, {"c": 2, "d": 4}], dict(), dict(),),
+        ([[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]], dict(), dict(),),
```

Comment:
Oops, I forgot about this before opening the PR draft. I haven't actually fixed the warning in this test yet. It comes from our function `assert_eq` when called with the list data on this line. The warning is raised when it [calls pandas](https://github.com/rapidsai/cudf/blob/2c6b0dac61a6671642bb5b076e910e20e2bdd1b6/python/cudf/cudf/testing/_utils.py#L112), which in turn calls numpy.

```python
DeprecationWarning: elementwise comparison failed; this will raise an error in the future.
```

The same issue was reported for arrow here: https://issues.apache.org/jira/browse/ARROW-13302

Should we silence this warning in our call to pandas and rely on pandas to fix it upstream? This seems out of scope for us.

Reply:
Yes, I think we should rely on pandas to fix this.

Reply:
Great. I silenced the warning in 99006cecf02b6040c78c3750d0de5b45145ad870.